python/lsst/ip/isr/isrFunctions.py

```diff
@@ -336,10 +336,23 @@ def overscanCorrection(ampMaskedImage, overscanImage, fitType='MEDIAN', order=1,
 
     numSigmaClip = statControl.getNumSigmaClip()
 
-    if fitType in ('MEAN', 'MEANCLIP', 'MEDIAN'):
+    if fitType in ('MEAN', 'MEANCLIP'):
         fitType = afwMath.stringToStatisticsProperty(fitType)
         offImage = afwMath.makeStatistics(overscanImage, fitType, statControl).getValue()
         overscanFit = offImage
+    if fitType in ('MEDIAN',):
+        # we need an image with integer pixels to handle ties properly
+        if hasattr(overscanImage, "image"):
+            imageI = overscanImage.image.convertI()
```

Comment:
Is it worth including a check that the data is indeed integer that has been converted to floating-point? Might someone want to run an overscan correction on an image that has floating-point values (maybe some other operation has been performed before the overscan correction)?

Reply:
I'll add an argument (default True) to allow the conversion to int.  But I don't think anyone would/should ever want to run overscan estimation on processed data.